src/effects/effects_base/view/effectsviewutils.cpp

```
@@ -0,0 +1,23 @@
+/*
+ * Audacity: A Digital Audio Editor
+ */
+#include "effectsviewutils.h"
+
+namespace au::effects {
+muse::uicomponents::MenuItemList utils::toMenuItemList(const std::vector<DropdownOption>& options, int selectedIndex)
+{
```

Cppcheck (2.20.0)

[information] Include file

(missingInclude)